### DISCUSSION TITLE
Event logs GET api not responding as expected

### DISCUSSION THREAD

**Discussion by surabhi-dudhediya** [walt-id/.github#91] [Help]
In the Wallet APIs, there is a GET api of event logs `/wallet-api/wallet/{wallet}/eventlog` which retrieves the event logs for currently signed in account wallet. This api has only one mandate query parameter as wallet id. Even after passing the correct wallet id to the api, the api is throwing below error
`{"exception":true,"status":"Bad Request","code":"400","message":"No such configuration registered: \"id.walt.webwallet.config.TrustConfig\"!"}`. Kindly provide me the solution for the same

**Comment by alegomes** [comment]
Hi @surabhi-dudhediya 

This forum is not active anymore. Can you please repost your question at the [waltid-identity/issues](https://github.com/walt-id/waltid-identity/issues)?

[]s

**Comment by surabhi-dudhediya** [comment]
ok
